src/metatrain/cli/train.py

```diff
@@ -98,6 +98,29 @@ def _prepare_train_model_args(args: argparse.Namespace) -> None:
     args.options = OmegaConf.merge(args.options, override_options)
 
 
+def _process_continue_from(continue_from: str) -> Optional[str]:
+    # covers the case where `continue_from` is `auto`
+    if continue_from == "auto":
```

Comment:
We should inform the user that the training is continued if the `outputs` directory is found or maybe error or warn if continue="auto" and now directory is found.

Reply:
Ok! It's now there